Neos.Neos/Classes/Fusion/Cache/GraphProjectorCatchUpHookForCacheFlushing.php

```diff
@@ -120,6 +155,27 @@ public function __construct(
     ) {
     }
 
+    public function canHandle(EventInterface $event): bool
```

Comment:
This is just a workaround, to reduce the subset of possible events, as we are not able to "detect" events, which provide a workspaceName. Also this brings a bit of clearity, which events this catchup hook actually expects.